cfgov/cfgov/settings/base.py

```diff
@@ -183,9 +183,9 @@
 # various locations.
 STATICFILES_FINDERS = (
     'sheerlike.finders.SheerlikeStaticFinder',
-    'transition_utilities.finders.NoPHPFileSystemFinder',
     'django.contrib.staticfiles.finders.AppDirectoriesFinder',
-    'compressor.finders.CompressorFinder',
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    #'compressor.finders.CompressorFinder',
```

Comment:
I re-enabled compressor-- I'm not sure we're using it, but disabling it is out of scope for this PR.